config/routes.rb

```diff
@@ -12,10 +12,10 @@
   resource :contractor, only: [:create, :show, :edit, :update] do
     resources :projects, only: [:create, :show, :index] do
       delete 'remove_dev', to: "projects#remove_dev"
-      get 'payment', to: "projects#submit_payment"
-      post 'payment', to: "projects#pay"
+      get 'payment', to: "projects#pay"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.